### PR TITLE
[YUNIKORN-1604] Add K8Shim state dump info

### PR DESCRIPTION
**Pull request by wilfred-s**
### What is this PR for?
K8Shim state dump information, high level only as content is not finalised yet.
Upgrade to Docusaurus 2.4.1

### What type of PR is it?
* [X] - Documentation

### What is the Jira issue?
https://issues.apache.org/jira/browse/YUNIKORN-1604

### How should this be tested?
Both en and zh-cn site builds should pass

### CODE REVIEW

**Comment by wilfred-s** [comment]
Requesting review in both languages before we commit...

**Review by craigcondit — APPROVED** [review]
+1 English side looks good.

**Review by 0yukali0 — CHANGES_REQUESTED** [review]
Almost done.

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/api/scheduler.md [hunk 1]
```diff
@@ -1404,6 +1405,13 @@ checksum: D75996C07D5167F41B33E27CCFAEF1D5C55BE3C00EE6526A7ABDF8435DB4078E
 
 该REST请求的输出将会很巨大，且为前面演示过的输出的组合。
 
+RM 诊断显示 K8Shim 缓存的内容。确切内容取决于版本并且尚未稳定。
+当前内容显示缓存的对象：
+* 节点
+* 豆荚
```
[the remark this answers is not in the document]
Comment (zhuqi-lucas) [reply i18n/zh-cn/docusaurus-plugin-content-docs/current/api/scheduler.md:1411]: +1 We just need to use Pods here.

[the remark this answers is not in the document]
Comment (wilfred-s) [reply i18n/zh-cn/docusaurus-plugin-content-docs/current/api/scheduler.md]: missed that one, I thought I had them all covered :-(

**Review by 0yukali0 — APPROVED** [review]
LGTM

**Review by zhuqi-lucas — APPROVED** [review]
LGTM

**Pull request merged**